sssd_test_framework/topology.py

```diff
@@ -213,3 +233,8 @@ def test_ldap(client: Client, provider: GenericProvider):
     """
     .. topology-mark:: KnownTopologyGroup.AnyAD
     """
+
+    IPATrust = [KnownTopology.IPATrustAD, KnownTopology.IPATrustSamba]
```

Comment:
To be consistent I'd name this topology `AnyIPATrust`

Reply:
If you change it, update the documentation as well

Reply:
Hmm. I don't think we need to follow "Any" as a specific pattern and I'm still trying to figure out if it is helpful here, I don't have problem with either. If it is just to "follow a pattern" I'd say there is not pattern and it is just and we don't require each name in KnownTopologyGroup to start with Any. If you have a reason, I'll change it.

AnyProvider is I think clear, "Any" has its own meaning. For AnyADProvider, I think we could have go with just "ADProvider" but "Any" kind of denotes that there are some limitations of what you can do (Samba is subset of AD). With IPA trust, you can only work with users and groups anyway and there are no limitations here since both AD and Samba behaves the same. So I guess that's why I didn't add "Any" here.

But I can go with either, it's your call.

Reply:
It's ok, I just wanted to raise that if we were following a pattern we should also follow it here. Your explanation is clear enough that there isn't any pattern to follow for this case